### PR TITLE
Punctuation Fix

### PR DESCRIPTION
**Pull request by Abdelrahman350**
I make this pull request to fix punctuation, grammar and typo mistakes.

### CODE REVIEW

**Comment by gf712** [comment]
Hey, thanks for the patch. But I wouldn't worry too much about the `NEWS` file

**Comment by Abdelrahman350** [comment]
You are welcome @gf712. Can this PR be merged?

File: doc/readme/DEVELOPING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This is a very basic checklist on how to get started with Shogun:
 2. Run the API examples, see [INTERFACES.md](INTERFACES.md), or create your own, see [EXAMPLES.md](EXAMPLES.md)
 4. Run the [tests](#testing).
 
-To avoid spending a lot of our time on explaining the same basic things, please **excessively** use the internet for any questions on the commands and tools needed.
+To avoid spending a lot of our time explaining the same basic things, please **excessively** use the internet for any questions on the commands and tools needed.
```
Comment (gf712) [inline doc/readme/DEVELOPING.md:9]: The original seems correct to me "spend time on..."

Comment (Abdelrahman350) [reply doc/readme/DEVELOPING.md:9]: For "spend" verb, if it is used to express activity, we don’t use a preposition, and we put the verb in the –ing form.

Comment (gf712) [reply doc/readme/DEVELOPING.md:9]: OK!

**Comment by karlnapf** [comment]
thanks!
Yeah as Gil said, no need to correct the NEWS file ....
As you have done a doc/readme PR now, what about sending in some code next? :)

**Comment by Abdelrahman350** [comment]
> thanks!
> Yeah as Gil said, no need to correct the NEWS file ....
> As you have done a doc/readme PR now, what about sending in some code next? :)

Yes, I have already created a pull request to send code in [https://github.com/shogun-toolbox/shogun/pull/4956] and I am waiting for your feedback @karlnapf.

**Pull request merged**